sql/driver/wire.go

```diff
@@ -62,50 +115,3 @@ func (Request) Method() Method {
 func (Request) CreateReply() Response {
 	return Response{}
 }
-
-// GetParameters returns the Params slice as a `parameters`.
```

Comment:
This code movement was part of Vivek's change. Unless there was some crucial urgency to doing this move, just wait until his change gets merged.